### ISSUE TITLE
NVDA rarely fails to open at the Windows login screen

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#17452]
### Steps to reproduce:
Press the power button, wait for Windows to start. In rare cases, NVDA will not start even though it is set to start at the Windows login screen.
### Actual behavior:

Windows starts, NVDA is set to start here, Windows starts properly but NVDA does not start.
### Expected behavior:

NVDA should start at the Windows login screen, which it does most of the time.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2024.4.1
#### Windows version:
Windows 11, 24 H2.
#### Name and version of other software used to reproduce the problem:
None.
#### Other information about your system:
None.
#### Does the problem persist after restarting your computer?
When the problem occurs varies, but restarting certainly does not fix the problem, which is rare.
#### Have you tried other versions of NVDA? If so, please report their behavior.
I have experienced this error with previous versions as well.
#### If the NVDA plugins are disabled, does your problem persist?
I am not using any plugins.
#### Does the problem persist after running the COM Registry Fix Tool from NVDA's tools menu?
It doesn't work

**Comment by haruncetinkaya306** [comment]
I reproduced this issue with issue #17453, there I was able to attach the logs correctly.
